lib/chat_api/conversations.ex

```diff
@@ -483,4 +503,31 @@ defmodule ChatApi.Conversations do
         dynamic
     end)
   end
+
+  @spec filter_pagination(map) :: Keyword.t()
+  def filter_pagination(args) do
+    Enum.reduce(
+      args,
+      [
+        # These are paginator options from https://hexdocs.pm/paginator/Paginator.html#c:paginate/3-options
+        cursor_fields: [{:messages, :inserted_at}, :id],
+        sort_direction: :desc,
+        fetch_cursor_value_fun: fn
+          schema, {:messages, :inserted_at} ->
+            # Get the `inserted_at` field of the most recent message
+            Enum.sort_by(schema.messages, & &1.inserted_at, :desc)
+            |> Enum.map(& &1.inserted_at)
```

Comment:
I have to take the most recent `message.inserted_at` for each conversation, so that i can use it for the cursor_field